src/components/synonyms/agSynonymTable.vue

```diff
@@ -323,22 +370,59 @@ export default {
         this.addAlert("Please load a substance", "warning");
         return;
       }
+      this.rowData.push(this.toRowData(null));
```

Comment:
```suggestion
      this.rowData.unshift(this.toRowData(null));
```
this will put the new synonym at the top of the table.

Reply:
Tested.  This functions well and is better for UX.